src/interfaces/reportInterfaces.ts

```diff
@@ -19,6 +19,47 @@ export type getForcedToFrameDataType = Promise<{
     groupRes: IGroup[];
     initialSelections: IPlanningRowSelections}>;
 
+export type getPlanningDataType = Promise<{
+  res: IProjectsResponse;
+    projects: IProject[];
+    classHierarchy: IClassHierarchy;
+    planningDistricts: {
+      districts: ILocation[];
+      year: number;
+      allLocations?: ILocation[];
+      divisions?: ILocation[];
+      subDivisions?: ILocation[];
+  } | {
+      districts: ILocation[];
+      allLocations: ILocation[];
+      divisions: ILocation[];
+      subDivisions: ILocation[];
+      year: number;
+  };
+    groupRes: IGroup[];
+    initialSelections: IPlanningRowSelections}>;
+
+export type IPlanningData = {
+  res: IProjectsResponse;
+  projects: IProject[];
+  classHierarchy: IClassHierarchy;
+  planningDistricts: {
+    districts: ILocation[];
+    year: number;
+    allLocations?: ILocation[];
+    divisions?: ILocation[];
+    subDivisions?: ILocation[];
+  } | {
+    districts: ILocation[];
+    allLocations: ILocation[];
+    divisions: ILocation[];
+    subDivisions: ILocation[];
+    year: number;
+  };
+  groupRes: IGroup[];
+  initialSelections: IPlanningRowSelections;
+}
```

Comment:
I would combine these two as they seem to have same properties. We could maybe leave only the IPlanningData and then use `Promise<IPlanningData>` in those places where we have a Promise with this type